### PR TITLE
[github.com/GoogleCloudPlatform/k8s-cloud-provider]:upgrade k8s.io/klog/v2

### PR DESCRIPTION
**Pull request by tanjunchen**
[github.com/GoogleCloudPlatform/k8s-cloud-provider]:upgrade k8s.io/klog/v2
ref:https://github.com/kubernetes/kubernetes/issues/89961

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -9,7 +9,7 @@ require (
 	golang.org/x/sys v0.0.0-20200116001909-b77594299b42 // indirect
 	google.golang.org/api v0.15.1-0.20200106000736-b8fc810ca6b5
 	google.golang.org/genproto v0.0.0-20200115191322-ca5a22157cba // indirect
-	k8s.io/klog v1.0.0
+	k8s.io/klog/v2 v2.0.0-rc.1
```
Comment (serathius) [inline go.mod:12]: Please change to v2.0.0

Comment (serathius) [inline go.mod:12]: Please change to v2.0.0

**Comment by serathius** [comment]
cc @bowei

File: go.mod (hunk 1 above)
Comment (tanjunchen) [reply go.mod:12]: sure . Thanks

**Comment by dims** [comment]
cc @rramkumar1

**Comment by bowei** [comment]
@spencerhance  can you take a look

**Pull request merged**